### ISSUE TITLE
Code generation fails with openapi v5 gen

### ISSUE THREAD

**Issue by harsimranmaan** [dart-ogurets/dart-openapi-maven#20]
https://github.com/dart-ogurets/dart-openapi-maven/blob/master/src/main/java/cd/connect/openapi/DartV3ApiGenerator.java#L192

```
Error: Exception in thread "main" java.lang.NoSuchFieldError: isMapContainer
	at cd.connect.openapi.DartV3ApiGenerator.lambda$updateAllModels$2(DartV3ApiGenerator.java:192)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1508)
	at cd.connect.openapi.DartV3ApiGenerator.lambda$updateAllModels$3(DartV3ApiGenerator.java:172)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1508)
	at cd.connect.openapi.DartV3ApiGenerator.lambda$updateAllModels$4(DartV3ApiGenerator.java:162)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at cd.connect.openapi.DartV3ApiGenerator.updateAllModels(DartV3ApiGenerator.java:158)
	at org.openapitools.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:504)
	at org.openapitools.codegen.DefaultGenerator.generate(DefaultGenerator.java:875)
	at org.openapitools.codegen.cmd.Generate.execute(Generate.java:432)
	at org.openapitools.codegen.cmd.OpenApiGeneratorCommand.run(OpenApiGeneratorCommand.java:32)
	at org.openapitools.codegen.OpenAPIGenerator.main(OpenAPIGenerator.java:66)

```
Full details at https://github.com/bigpanther/trober/pull/92/checks?check_run_id=1614823203
isMapContainer has been removed in https://github.com/OpenAPITools/openapi-generator/pull/7747

Causes: https://github.com/gibahjoe/openapi-generator-dart/issues/27

**Comment by rvowles** [comment]
Version 5 is now out, so I can fix it.

**Comment by rvowles** [comment]
so isMapContainer and isListContainer have been renamed, to isMap and isArray

**Comment by rvowles** [comment]
I might add in null safety support with this release

**Comment by rvowles** [comment]
I will delay null safety to the next release. This is now released as 4.1
